x-pack/test_serverless/functional/config.base.ts

```diff
@@ -5,17 +5,29 @@
  * 2.0.
```

Comment:
I think this feature could be useful for API integration testing as well, so I'd suggest to move it from `x-pack/test_serverless/functional` to `x-pack/test_serverless/shared` (where it would be available for both, functional and API integration tests).

Reply:
Addressed in [00ca617](https://github.com/elastic/kibana/pull/192064/commits/00ca61738faa1ff6e6e9591988f7384d0083933c).